modules/xfem/doc/content/source/userobjects/MeshCut2DFractureUserObject.md

```diff
@@ -4,19 +4,31 @@
+This class is used to define an evolving cutting plane for 2D XFEM simulations based on a mesh that defines the initial crack and uses fracture integrals or maximum stress to grow that crack. It (1) reads in a mesh describing the crack surface or uses nucleated cracks, (2) uses the mesh to do initial cutting of 2D elements, and (3) grows the mesh incrementally based on fracture domain integrals to allow nonplanar crack growth based on propagation directions determined by fracture integrals.  A maximum stress criterion can also be used to drive crack growth.
 
 The crack propagates if the failure criterion is met, given by:
+
 \begin{equation}
 K_c \le \sqrt{K^2_I+K^2_{II}}
 \end{equation}
-where $K_I$ and $K_{II}$ are the mode I and II stress intensity factors provided by the fracture integral and the material property $K_c$ is given defined in the input file by [!param](/UserObjects/MeshCut2DFractureUserObject/k_critical).  The crack growth direction is given by the direction the maximized the crack-tip hoop stress, given by Equation 5 in [!cite](jiang2020).  The growth increment is a user provided input given in the defined by [!param](/UserObjects/MeshCut2DFractureUserObject/growth_increment).
+
+where $K_I$ and $K_{II}$ are the mode I and II stress intensity factors provided by the fracture integral and the material property $K_c$ is given defined in the input file by [!param](/UserObjects/MeshCut2DFractureUserObject/k_critical).  The crack growth direction is given by the direction the maximized the crack-tip hoop stress, given by Equation 5 in [!cite](jiang2020).  The growth increment is a user provided input given in the defined by [!param](/UserObjects/MeshCut2DFractureUserObject/growth_increment).  The fracture integrals $K_I$ and $K_{II}$ are obtained from the [InteractionIntegral.md] vectorpostprocessor specified in the input file by [!param](/UserObjects/MeshCut2DFractureUserObject/ki_vectorpostprocessor) and [!param](/UserObjects/MeshCut2DFractureUserObject/kii_vectorpostprocessor).  Defaults for [!param](/UserObjects/MeshCut2DFractureUserObject/ki_vectorpostprocessor) and [!param](/UserObjects/MeshCut2DFractureUserObject/kii_vectorpostprocessor) use the names produced by the [/DomainIntegralAction.md] which is the standard way to set-up the [InteractionIntegral.md].
+
+Near a free surface, the integration volume of the q-function used to compute the `InteractionIntegral` will intersect the surface, leading to a reduction in the fracture integral values.  This can lead to cracks becoming stuck near free surfaces.  For these cases, a maximum stress criterion computed using [CrackFrontNormalStress.md] vectorpostprocessor can be used for crack growth using an additional failure criterion given by:
```

Comment:
```suggestion
Near a free surface, the integration volumes of the rings used to compute the `InteractionIntegral` will intersect the surface, leading to a reduction in the fracture integral values.  This can lead to cracks becoming unable to farther propagate as they approach free surfaces.  For these cases, a maximum stress criterion computed using [CrackFrontNormalStress.md] vectorpostprocessor can be used for crack growth using an additional failure criterion given by:
```